ksql-engine/src/test/java/io/confluent/ksql/EndToEndEngineTestUtil.java

```diff
@@ -189,6 +189,9 @@ public void configure(final Map<String, ?> properties, final boolean b) {
 
     @Override
     public Object deserialize(final String topicName, final byte[] data) {
+      if (data == null) {
```

Comment:
Were all of these null checks added due to the new tests failing?

Reply:
Yep